python/descriptor_pool.c

```diff
@@ -159,26 +159,39 @@ static bool PyUpb_DescriptorPool_TryLoadFilename(PyUpb_DescriptorPool* self,
 }
 
 bool PyUpb_DescriptorPool_CheckNoDatabase(PyObject* _self) {
-  PyUpb_DescriptorPool* self = (PyUpb_DescriptorPool*)_self;
-  if (self->db) {
-    PyErr_SetString(
-        PyExc_ValueError,
-        "Cannot call Add on a DescriptorPool that uses a DescriptorDatabase. "
-        "Add your file to the underlying database.");
-    return false;
+  return true;
+}
+
+static bool PyUpb_DescriptorPool_LoadDependentFiles(
+    PyUpb_DescriptorPool* self, google_protobuf_FileDescriptorProto* proto) {
+  // Load dependent files if necessary.
```

Comment:
this comment now simply restates the method name, you can drop it

Reply:
Done.